src/libs/API.js

```diff
@@ -54,11 +54,15 @@ function addAuthTokenToParameters(command, parameters) {
         // If we end up here with no authToken it means we are trying to make an API request before we are signed in.
         // In this case, we should cancel the current request by pausing the queue and clearing the remaining requests.
         if (!authToken) {
-            redirectToSignIn();
-
             console.debug('A request was made without an authToken', {command, parameters});
             Network.pauseRequestQueue();
             Network.clearRequestQueue();
+
+            redirectToSignIn().finally(() => {
```

Comment:
It might be good to add some context here about why we are unpausing the queue. Something like...

>We are unpausing the queue after we finish signing in so that new requests can be made again.